crates/extensions/tedge_mqtt_bridge/src/lib.rs

```diff
@@ -400,9 +453,23 @@ async fn half_bridge(
                         None => break,
                     }
                 } else {
-                    info!("Bridge cloud connection {name} ignoring already known pkid={pkid}");
+                    info!("Bridge {name} connection ignoring already known pkid={pkid}");
+                    ignored += 1;
                 }
             }
+
+            Event::Outgoing(Outgoing::AwaitAck(pkid)) => {
+                info!("Bridge {name} connection still waiting ack for pkid={pkid}");
+            }
+
+            Event::Incoming(Incoming::Disconnect) => {
+                info!("Bridge {name} connection closed by peer");
+            }
+
+            Event::Outgoing(Outgoing::PingReq) => {
+                let waiting = forward_pkid_to_received_msg.len();
+                info!("Bridge {name} connection: received={received} forwarded={forwarded} published={published} waiting={waiting} acknowledged={acknowledged} finalized={finalized} ignored={ignored}");
```

Comment:
Is this meant to be at INFO level?

Reply:
It is. Except that I wonder if this appropriate to log this progress every minute. Every hours might be useful.

My only concern here is that this is not logged when the builtin bridge is blocked! i.e. when this would be the more insightful :-(

Reply:
I will remove this not so usefull info! message.

Reply:
Removed: https://github.com/thin-edge/thin-edge.io/pull/3122/commits/4cadd6dd24a5ece70b21a84dca9a0cef681c110e